ports/espressif/boards/lilygo_ttgo_t8_s2_st7789/board.c

```diff
@@ -41,7 +41,7 @@ uint8_t display_init_sequence[] = {
     // normal display mode on
     0x13, 0,
     // display and color format settings
-    0x36, 1, 0x08,
+    0x36, 1, 0x68,
```

Comment:
This changes D5 MV Page/Column Order to one (reverse mode) and D6 MX Column Address Order to one (now Right to Left instead of Left to Right) in the ST7789 configuration. Effectively rotating the display to the landscape mode we are using it with refresh from top to bottom when writing/updating.